integration/tests/lib/receipt_state.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-const receiptTimeout = 60 * time.Second
+const receiptTimeout = 120 * time.Second
```

Comment:
same question as for [integration/tests/lib/block_state.go](https://github.com/onflow/flow-go/pull/2599/files#r901761157) - why the increase in the timeout?

Reply:
Please find my answer to the previous comment.